internal/style/style_test.go

```diff
@@ -0,0 +1,39 @@
+package style_test
+
+import (
+	"testing"
+
+	"github.com/heroku/color"
+	"github.com/sclevine/spec"
+	"github.com/sclevine/spec/report"
+
+	"github.com/buildpacks/pack/internal/style"
+	h "github.com/buildpacks/pack/testhelpers"
+)
+
+func TestStyle(t *testing.T) {
+	color.Disable(true)
+	defer color.Disable(false)
+	spec.Run(t, "testStyle", testStyle, spec.Parallel(), spec.Report(report.Terminal{}))
+}
+
+func testStyle(t *testing.T, when spec.G, it spec.S) {
+	when("#Style", func() {
```

Comment:
```suggestion
	when("#Symbol", func() {
```
Typically, we try to have separate `when` blocks for funcs. Similarly, I'd recommend making a separate `when` for `#Map`